### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string.

### ISSUE THREAD

**Issue by fxmiko** [open-wa/wa-automate-nodejs#3015]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Legacy (Normal)

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Current Behavior

Wont run normally, shows puppeteer error after running then closes.

### Expected Behavior

Should Run Like Normal

### Steps To Reproduce

It crashed by just running it.

i named the code below as "index.js"
i run it with "node index"
then crashed

### create() code OR full CLI command + CONFIG

```markdown
const wa = require('@open-wa/wa-automate');

wa.create({
  sessionId: "COVID_HELPER",
  multiDevice: true, //required to enable multiDevice support
  authTimeout: 60, //wait only 60 seconds to get a connection with the host account device
  blockCrashLogs: true,
  disableSpins: true,
  headless: true,
  useChrome: true,
  browserRevision : '1069273',
  hostNotificationLang: 'PT_BR',
  logConsole: false,
  popup: true,
  qrTimeout: 0, //0 means it will wait forever for you to scan the qr code
}).then(client => start(client));

function start(client) {
  client.onMessage(async message => {
    if (message.body === 'zzx') {
    try{
            
    //   await client.sendText(message.from, '👋 Hello!');
      await client.sendFileFromUrl(message.from, "https://video.twimg.com/tweet_video/Fkze4LvaEAAK21M.mp4", 'twtmp4.mp4', ``, message.id)
         
    } catch (err) {
            console.log(err)
        } 
    }
  });
}
```

### DEBUG INFO

```markdown
It crashed before even showing that, crashed right after - Launching Browser
```

### Environment

```markdown
OS : Ubuntu 18.04.6 LTS x86_64
Node Version: v16.19.0
NPM Version: 9.3.0
```

### Screenshots/Logs
```
info: Set up logging for file  {"timestamp":"2023-01-16T12:01:36.433Z"}
info: popup port set to 7000 {"timestamp":"2023-01-16T12:01:36.988Z"}
info: You can also authenticate the session at: http://localhost:7000 {"timestamp":"2023-01-16T12:01:37.045Z"}
info: RAM INFO Total: 8.36 GB | Free: 7.01 GB {"timestamp":"2023-01-16T12:01:37.048Z"}
info: PPTR VERSION INFO 19.5.2 {"timestamp":"2023-01-16T12:01:37.061Z"}
info: Starting  {"data":"Starting","eventName":"STARTUP.COVID_HELPER","eventNamespace":"STARTUP","sessionId":"COVID_HELPER","timestamp":"2023-01-16T12:01:37.087Z"}
info: Version: 4.55.0  {"data":"Version: 4.55.0","eventName":"STARTUP.COVID_HELPER","eventNamespace":"STARTUP","sessionId":"COVID_HELPER","timestamp":"2023-01-16T12:01:37.089Z"}
info: Initializing WA  {"data":"Initializing WA","eventName":"STARTUP.COVID_HELPER","eventNamespace":"STARTUP","sessionId":"COVID_HELPER","timestamp":"2023-01-16T12:01:37.090Z"}
info: SCRIPT READY: jsSha.min.js 8982 {"timestamp":"2023-01-16T12:01:37.116Z"}
info: SCRIPT READY: qr.min.js 18603 {"timestamp":"2023-01-16T12:01:37.118Z"}
info: SCRIPT READY: base64.js 4173 {"timestamp":"2023-01-16T12:01:37.118Z"}
info: SCRIPT READY: hash.js 34983 {"timestamp":"2023-01-16T12:01:37.119Z"}
info: SCRIPT READY: wapi.js 79911 {"timestamp":"2023-01-16T12:01:37.119Z"}
info: SCRIPT READY: launch.js 65074 {"timestamp":"2023-01-16T12:01:37.120Z"}
info: Launching Browser  {"data":"Launching Browser","eventName":"STARTUP.COVID_HELPER","eventNamespace":"STARTUP","sessionId":"COVID_HELPER","timestamp":"2023-01-16T12:01:37.121Z"}
info: The "path" argument must be of type string. Received undefined  {"data":"The \"path\" argument must be of type string. Received undefined","eventName":"STARTUP.COVID_HELPER","eventNamespace":"STARTUP","sessionId":"COVID_HELPER","timestamp":"2023-01-16T12:01:37.132Z"}
```
![Qk6LldVSAt0AAAAASUVORK5CYII=](https://user-images.githubusercontent.com/43951246/212674145-527e29cf-6814-4d18-8cb9-ce36a4b254c6.png)

### Anything else?

This does not happen in version 4.53.0 only happen on version 4.54.0 or newer after updating dependencies

**Comment by smashah** [comment]
looks like there's an issue with the `browserRevision` config.

You can't `useChrome: true` and `browserRevision` at the same time. `browserRevision` tells the process to go and download that version of chromium. `useChrome: true` will try to find the local install of google chrome (different from chromium).

I suggest removing `browserRevision`

**Comment by fxmiko** [comment]
same error after deleting `useChrome: true`, deleting `browserRevision` gives me another error, 

````
Error: Failed to launch the browser process!


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/home/adept/my-projects/newneko/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:299:20)
    at ChildProcess.<anonymous> (/home/adept/my-projects/newneko/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:290:24)
    at ChildProcess.emit (node:events:525:35)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: Failed to launch the browser process!


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/home/adept/my-projects/newneko/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:299:20)
    at ChildProcess.<anonymous> (/home/adept/my-projects/newneko/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:290:24)
    at ChildProcess.emit (node:events:525:35)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
````

I'm using Google Chrome 85.0.4183.121

**Comment by smashah** [comment]
You have to use version 106+.

Please update your google chrome.

**Comment by fxmiko** [comment]
will try when my administrator installs the version 106+, will keep this updated

**Comment by fxmiko** [comment]
**Solved using latest Google Chrome 109.0.5414.74**

Just make sure `headless` set to `true` and  add `--no-sandbox` to `chromiumArgs`

Here's a copy of my code i used to test, **just in case** anyone has the same error as mine
```
const wa = require('@open-wa/wa-automate');

wa.create({
  sessionId: "COVID_HELPER",
  multiDevice: false, //required to enable multiDevice support
  authTimeout: 60, //wait only 60 seconds to get a connection with the host account device
  blockCrashLogs: true,
  disableSpins: true,
  headless: true,
  useChrome: true,
  // browserRevision : '1069273',
  executablePath: '/usr/bin/google-chrome-stable',
  hostNotificationLang: 'PT_BR',
  logConsole: false,
  // "logging":[{"type":"file"}],
  popup: true,
  qrTimeout: 0, //0 means it will wait forever for you to scan the qr code
  chromiumArgs: [
  '--no-sandbox',
  ],
}).then(client => start(client));

function start(client) {
  client.onMessage(async message => {
    if (message.body === 't') {
    try{
            
    //   await client.sendText(message.from, '👋 Hello!');
      await client.sendFileFromUrl(message.from, "https://video.twimg.com/tweet_video/Fkze4LvaEAAK21M.mp4", 'twtmp4.mp4', ``, message.id)
         
    } catch (err) {
            console.log(err)
        } 
    }
  });
}
```
